### ISSUE TITLE
Interface 'Stringable' not found

### ISSUE THREAD

**Issue by encreinformatique** [stayallive/wp-sentry#107]
Hello, this morning, a Wordpress got down after updating the plugin to the last version.
The error is with sentry in the vendor folder. If I am remember well, the Stringable interface was introduced in PHP 8.0.

```
PHP message: PHP Fatal error:  Uncaught Error: Interface 'Stringable' not found in /var/www/paroledevelo.com/web/wp-content/plugins/wp-sentry-integration/build/vendor/sentry/sentry/src/Dsn.php:12
```

PHP : 7.4.25
Wordpress : 5.8.2
Plugin : 4.10.0

**Comment by stayallive** [comment]
That isn't right obviously, apologies for the errors, I'll have a look on why this interface is added and why that's on non-8 installations.

**Comment by stayallive** [comment]
This should be fixed in [v4.10.1](https://github.com/stayallive/wp-sentry/releases/tag/v4.10.1) that was just released.
